analyzers/src/SonarAnalyzer.CSharp/Helpers/MessageTemplateExtractor.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
```

Comment:
I think this should go into MessageTemplates folder.

Reply:
I also will move the rule itself, since the scaffolding script emits it at top `Rules/` level.